### DISCUSSION TITLE
Tried rsuite, but responsive it's not so 100% good.

### DISCUSSION THREAD

**Discussion by vitalijalbu** [rsuite/rsuite#2880] [General]
Hi guys, how cvome all compoennts are great but the responsive part is not so good? Yo ucan see a preview: https://rsuitejs.com/resources/templates/

<img width="757" alt="Schermata 2022-11-08 alle 12 24 36" src="https://user-images.githubusercontent.com/40239902/200552301-97af6609-5438-4f52-b51f-e3cbfbb87f53.png">

**Comment by SevenOutman** [comment]
Hi @vitalijalbu, "React Suite is designed and implemented for use on modern desktop browsers rather than mobile browsers", quoting the README.

**Reply by vitalijalbu** [reply]
yes, agree with you but nowdays 80% of visitors come from mobile devices.

**Reply by vitalijalbu** [reply]
what do you think are the weaknesses of rsuitejs? if only the grid then it's not a problem, if I can contribute to make it better then it's not a problem. One more thing, for a custom theme I should overwrite the default theme? Or I can create a blank .scss file and define the rsuite theme variables? Thanks in advance.
